config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  get 'tutorials/new'
```

Comment:
Because this is `new` and `index` you could do `resources` and then `only` the specific actions you want. It makes it a bit less verbose in a larger routes file